test/cypress/integration/04-collective.test.js

```diff
@@ -162,6 +164,43 @@ describe('Collective page', () => {
       cy.get('[data-cy="longDescription"]').should('have.html', '<div><strong>Hello world!</strong></div>');
     });
   });
+
+  describe('Add funds modal', () => {
```

Comment:
I would rather add this in `test/cypress/integration/29-host.dashboard.test.js` and test the feature from the host dashboard. You'll find that tests are already setup there with a host admin account, which removes the need to add [this exception](https://github.com/opencollective/opencollective-api/pull/5402#discussion_r618355511) in the API.

Reply:
@Betree : The host dashboard doesn't have the actions dropdown if I am not mistaken. Therefore we cannot access the add fund modal there. Am I missing something? 🤔

Reply:
There's no actions dropdown, but the add funds modal has its own button there: 
![image](https://user-images.githubusercontent.com/1556356/116317433-9e321900-a7b3-11eb-9197-2e1c039f752b.png)

Reply:
Ah, thanks for the explanation. That makes sense now. 😄